src/test/scala/nl/knaw/dans/easy/multideposit/actions/AddFileMetadataToDepositSpec.scala

```diff
@@ -72,53 +74,92 @@ class AddFileMetadataToDepositSpec extends UnitSpec with BeforeAndAfter with Bef
   }
 
   "datasetToFileXml" should "produce the xml for all the files" in {
-    AddFileMetadataToDeposit.datasetToFileXml("ruimtereis01").child.map(Utility.trim) should (
-      have length 5 and
-        contain allOf(
-      <file filepath="data/ruimtereis01_verklaring.txt"><dcterms:format>text/plain</dcterms:format></file>,
-      <file filepath="data/reisverslag/deel01.docx"><dcterms:format>application/vnd.openxmlformats-officedocument.wordprocessingml.document</dcterms:format></file>,
-      <file filepath="data/reisverslag/deel01.txt"><dcterms:format>text/plain</dcterms:format></file>,
-      <file filepath="data/reisverslag/deel02.txt"><dcterms:format>text/plain</dcterms:format></file>,
-      <file filepath="data/reisverslag/deel03.txt"><dcterms:format>text/plain</dcterms:format></file>
-    ))
-  }
-
-  //file does not need to exist for the mimetype to be established. it is based solely on filename.
+    inside(AddFileMetadataToDeposit.datasetToFileXml("ruimtereis01")) {
+      case Success(xml) => xml.child.map(Utility.trim) should (
```

Comment:
The return type of `datasetToFileXml` changed from `Elem` to `Try[Elem]`. That's why I had to make this change.